docker-compose.yml

```
@@ -34,7 +34,7 @@ services:
     restart: always
 
   grafana:
-    image: grafana/grafana-oss:${GRAFANA_VERSION:-latest}
+    image: grafana/grafana-oss:${GRAFANA_VERSION:-9.5.3}
```

```suggestion
    image: grafana/grafana-oss:${GRAFANA_VERSION:-10.1.0}
```

I am not super worried about being on the absolute latest for these images, but I think `10.x` has been out for awhile....

---

Nevermind. Just saw https://github.com/medic/cht-watchdog/pull/85